go.mod

```diff
@@ -542,3 +542,7 @@ require (
 )
 
 replace github.com/ma314smith/signedxml v1.1.1 => github.com/moov-io/signedxml v1.1.1
+
```

Comment:
Info on [forked repo tag](https://github.com/peteski22/go-proxyproto/releases/tag/v1.0.0).

> Keep listener listening if upstream connection address is not trusted (but close that connection).
> Releases the changes requested on pires's go-proxyproto by https://github.com/pires/go-proxyproto/pull/110.

Bumped to v1.0.0 which doesn't exist as a tag on the maintainer's repo but does on the forked repo.